### PR TITLE
Improve ockam relay list --help documentation

### LINKED ISSUE #6297: Improve `ockam relay list --help` by mentioning it works w/ Orchestrator enrollment by default
# Current behavior

When a user runs `ockam relay list --help` the example in the help text can be improved. This is show in this screenshot.

Image

# Desired behaviors

1) The help text for the command says `List Relays`, leave that as is (this is the short help text). Add a long help text that says: `List Relays on your default node. If you pass '--to <NODE>' then it lists the Relays at the given node`

2) Change the help for the `--to` argument to be: `Get the list of Relays at the given node`

3) The help text example says to run ~~`$ ockam relay list --at n2`~~
Instead, it should say `$ ockam relay list --to n2`

# Implementation details

> Here's an example of using the `long_about` attribute in another file:
> 
> https://github.com/build-trust/ockam/blob/59f52a66cc84ad682d9f4ea5715057f97f8eb7a3/implementations/rust/ockam/ockam_command/src/node/create.rs#L50-L52

Please take a look at this [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/relay/list.rs#L26) to get started.

---
<i>

Some tips for newcomers:
1. Create a new branch for the issue you are working on (don't push directly to your fork's `develop` branch)
1. In the PR description, include the issue that's being solved 
1. Run `cargo fmt`. You can configure the editor to run it automatically every time a file is saved
1. Run `cargo clippy` to make sure the code compiles + follows rust standards/best practices
1. You don't need to keep your branch updated with `develop` until there is a merge conflict. If you still want to update it, use `git rebase develop` instead of `git merge develop` 

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).
</i>

### PR DESCRIPTION
**Pull request by 0scvr**
## Current behavior

See #6297

## Proposed changes

Updated `ockam relay list`'s long about description, example and `--to` argument description.

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

Answers #6297.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/relay/list.rs [hunk 1]
```diff
@@ -18,17 +18,21 @@ use crate::util::node_rpc;
 use crate::{docs, CommandGlobalOpts};
 
 const PREVIEW_TAG: &str = include_str!("../static/preview_tag.txt");
+const LONG_ABOUT: &str = include_str!("./static/list/long_about.txt");
 const AFTER_LONG_HELP: &str = include_str!("./static/list/after_long_help.txt");
 
 /// List Relays
+///
+/// List Relays on your default node. If you pass '--to <NODE>' then it lists the Relays at the given node
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/relay/list.rs]: I think these two lines should be deleted right? This piece of text is the one loaded in the `long_about = docs::about(LONG_ABOUT),` line.

File: implementations/rust/ockam/ockam_command/src/relay/list.rs [hunk 2]
```diff
@@ -18,17 +18,21 @@ use crate::util::node_rpc;
...
 #[derive(Clone, Debug, Args)]
 #[command(
     arg_required_else_help = false,
     before_help = docs::before_help(PREVIEW_TAG),
+    long_about = docs::about(LONG_ABOUT),
     after_long_help = docs::after_help(AFTER_LONG_HELP)
 )]
 pub struct ListCommand {
-    ///  List all the relays relaying traffic to the specified node
+    ///  Get the list of Relays at the given node
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/relay/list.rs]: While we are at it, could you remove the extra space at the beginning?

File: implementations/rust/ockam/ockam_command/src/relay/list.rs (hunk 1 above)
Comment (0scvr) [reply implementations/rust/ockam/ockam_command/src/relay/list.rs]: Yes, totally! I forgot to remove these lines after adding long_about instead.

File: implementations/rust/ockam/ockam_command/src/relay/list.rs (hunk 2 above)
Comment (0scvr) [reply implementations/rust/ockam/ockam_command/src/relay/list.rs]: Sure thing 😏

**Review by adrianbenavides — DISMISSED** [review]
Awesome, LGTM! 🚀 

Thank you @0scvr 🙏

**Review by etorreborre — APPROVED** [review]

**Pull request merged**